SofaKernel/modules/SofaBaseTopology/src/SofaBaseTopology/EdgeSetGeometryAlgorithms.inl

```diff
@@ -932,7 +932,7 @@ bool EdgeSetGeometryAlgorithms<DataTypes>::computeEdgeSegmentIntersection(EdgeID
 
     Real deno, num;
     deno = d1010 * dbaba - dba10 * dba10;
-    if (!abs(deno) <= EPS)
+    if (!(abs(deno) <= EPS))
```

Comment:
I don't ask this change because it's equivalent, but usually, I tend to write `abs(deno) > EPS` instead of `!(abs(deno) <= EPS)`

Reply:
Is there a particular reason to keep it like this? :thinking:

Reply:
Indeed, no.